### PR TITLE
Webex Meetings integration blog post

### PR DESCRIPTION
**Pull request by dwelle**
wip

### CODE REVIEW

File: content/blog/webex-integration/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+<!-- end -->
+
+## Excalidraw ❤️ Meetings
+
+Listening to your stories it's amazing what you use Excalidraw for in the wild, from illustrating blog posts, to wireframing, teaching, presenting, to interviewing. One common denominator is collaboration, which has always been a core part of Excalidraw. Often times, text and drawings aren't enough, and for this purpose we're excited to anounce Excalidraw integration for Webex Meetings.
+
+## Excalidraw in Webex
+
+<video src="./webex-blog-promo.mp4" autoplay playsinline loop muted style="width: 100%; height: auto;"></video>
```
[the remark this answers is not in the document]
Comment (dwelle) [reply content/blog/webex-integration/index.md:19]: I think it's best to use a shorter and fresh video for the blog post. True for the link

File: content/blog/webex-integration/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: Introducing Webex Integration
+date: 2021-10-28
+author: Aakansha, David
+link: https://twitter.com/aakansha1216, https://twitter.com/dluzar
```
Comment (ad1992) [inline content/blog/webex-integration/index.md:5]: These links are not working in https://excalidraw-blog-git-webex-excalidraw.vercel.app/!  its joining both the links :p

Comment (dwelle) [reply content/blog/webex-integration/index.md:5]: oh, forgot about LP

File: content/blog/webex-integration/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+image: webex.png
```
Comment (ad1992) [inline content/blog/webex-integration/index.md:6]: whats use of this ? where does it show up ?

Comment (dwelle) [reply content/blog/webex-integration/index.md:6]: forgot about actually adding it — it's meant for OG

**Review by ad1992 — APPROVED** [review]

**Pull request merged**